CMakeLists.txt

```diff
@@ -284,7 +284,8 @@ endif()
 #===============================================================================
 
 add_subdirectory(vendor/pybind11)
-pybind11_add_module(openmc_cpp openmc/cppapi/cppapi.cpp)
+file(GLOB_RECURSE OPENMC_SOURCES src/cppapi/**.cpp)
```

Comment:
The CMake documentation actually recommends against using globs to find source files (see note [here](https://cmake.org/cmake/help/latest/command/file.html#filesystem)).